alica_engine/include/engine/BasicBehaviour.h

```diff
@@ -33,88 +33,65 @@ struct BehaviourContext
 /**
  * The base class for all behaviours. All Behaviours must inherit from this class.
  */
-class BasicBehaviour : private RunnableObject
+class BasicBehaviour : private RunnableObjectNew
```

Comment:
I guess this name is temporary until Planpool can use the same object?

Reply:
Yes, will change in the 3rd cleanup pr after PlanPool.